### PR TITLE
Fixes for guides

### PR DESCRIPTION
**Pull request by adborden**
Link to missing guides, fix some formatting.

### CODE REVIEW

File: pages/tools.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ secondary_nav:
 ---
 ## Agency data inventories
 
+[How to get your Open Data on Data.gov]({{ site.baseur }}{% link tools/how-to-get-your-open-data-on-datagov.md %}).
```
Comment (mogul) [inline pages/tools.md]: Should be site.baseurl, right?

Comment (adborden) [reply pages/tools.md]: :frowning_face: added an issue to improve testing here...

Comment (adborden) [reply pages/tools.md]: Pushed a fix.

**Comment by mogul** [comment]
Peer review caught it before it reached prod so the system as a whole works at least.

**Pull request merged**